config/services.yml

```diff
@@ -412,6 +412,36 @@ https://cmr.earthdata.nasa.gov:
       - image: !Env ${TRAJECTORY_SUBSETTER_IMAGE}
         operations: ['variableSubset', 'spatialSubset', 'shapefileSubset']
 
+  - name: sds/HOSS-maskfill
```

Comment:
This entry is almost exactly copied from the UAT one with the following two differences:

* A different UMM-S reference.
* Not including a UAT test collection under the `collections` attribute for the service.